lib/node_modules/@stdlib/blas/ext/base/dapxsumors/manifest.json

```diff
@@ -34,7 +34,12 @@
 				"-lm"
 			],
 			"libpath": [],
-			"dependencies": []
+			"dependencies": [
+				"stdlib/napi/export.h",
+				"stdlib/napi/argv.h",
+				"stdlib/napi/argv_int64.h",
+      			"stdlib/napi/argv_strided_float64array.h"
+			]
 		}
 	]
```

Comment:
The indentation in the `manifest.json` file should be changed to 2-spaces and configurations for the examples and benchmarks should be added, see the linked PRs.